doc/ref_model/chapters/chapter07.md

```diff
@@ -440,20 +440,21 @@ Insuring that the security standards and best practices are incorporated into th
 
 |  Ref | Requirement  | Definition/Note  |
 |-------|------|-------|
-| req.sec.gen.001 | The Platform **must** maintain the state to what it is specified to be and does not change unless through change management process |  |
-| req.sec.gen.002 | All systems part of Cloud Infrastructure **must** support password hardening (strength and rules for updates (process), storage and transmission, etc.) | Hardening: NIST SP 800-63B |
-| req.sec.gen.003 | All servers part of Cloud Infrastructure **must** support a root of trust and secure boot |  |
-| req.sec.gen.004 | The Operating Systems of all the servers part of Cloud Infrastructure **must** be hardened | NIST SP 800-123 |
-| req.sec.gen.005 | The Platform **must** support Operating System level access control | Details on OS |
-| req.sec.gen.006 | The Platform **must** support Secure logging. Logging with root account must be prohibited when root privileges are not required | Details |
-| req.sec.gen.007 | All servers part of Cloud Infrastructure **must** be Time synchronized with authenticated Time service | |
-| req.sec.gen.008 | All servers part of Cloud Infrastructure **must** be regularly updated to address security vulnerabilities | |
-| req.sec.gen.009 | The Platform **must** support Software integrity protection and verification | |
-| req.sec.gen.010 | The Cloud Infrastructure **must** support Secure storage (all types) | Expand/Delete based on other requirements |
-| req.sec.gen.011 | The Cloud Infrastructure **should** support Read and Write only storage partitions (write only permission to one or more authorized actors) | |
-| req.sec.gen.012 | The Operator **must** ensure that only authorized actors have physical access to the underlying infrastructure | |
-| req.sec.gen.013 | The Platform **must** ensure that only authorized actors have logical access to the underlying infrastructure | |
-| req.sec.gen.014 | All servers part of Cloud Infrastructure **should** support measured boot and an attestation server that monitors the measurements of the servers | |
+| req.sec.gen.001 | The Platform **must** maintain the specified configuration. |  |
+| req.sec.gen.002 | All systems part of Cloud Infrastructure **must** support password hardening as defined in [CIS Password Policy Guide]https://www.cisecurity.org/white-papers/cis-password-policy-guide/. | Hardening: CIS Password Policy Guide |
```

Comment:
@tomkivlin, @zwarico, @pgoyal01: I have tried and it is working for me.
@tomkivlin, can you please review and approve it you can? I would like to merge these good updates into the RM Master.

@zwarico, could you start looking at other sections of Chapter 7 Requirements and try to clean them up and improve?  See also my review of this Section in https://wiki.lfnetworking.org/display/LN/2020-08-26+-+%5BCNTT+-+RM+Workstream+Master%5D+Agenda+and+Meeting+Minutes

Reply:
@tomkivlin Yes it is better for it to be in parentheses so as to work as a link (related to the text).

@zwarico Can you  please add parentheses around the url "https://www.cisecurity.org/white-papers/cis-password-policy-guide/" with no space between the preceding "]" so that the text appears as in:
| req.sec.gen.002 | All systems part of Cloud Infrastructure **must** support password hardening as defined in [CIS Password Policy Guide](https://www.cisecurity.org/white-papers/cis-password-policy-guide/). | Hardening: CIS Password Policy Guide |